ci/computecpp.filter

```diff
@@ -2,40 +2,30 @@ accessor
 address_space
```

Comment:
Does this mean that all of the remaining tests are failing with ComputeCpp? That's a very long list, is that because of missing SYCL 2020 features?